docs/PersonalApplications.rst

```diff
@@ -334,33 +334,53 @@ via the following topics:
  * :ref:`Accepting/Denying a connection request <Accept-Deny-Connection-Request>`
  * :ref:`Disconnecting from your contacts <Disconnect-Contact>`
 
-To build your connections, you first need to click your display name on
-the top navigation bar and select My Connections from the drop-down
-menu.
+To build your connections, you first need to click the humberger menu on
+the side navigation bar and select **People**.
 
-|image24|
+|PEOPLE1|
 
-The **My Connections** page appears.
+The **People** page appears.
 
     .. note:: If there are so many connections, the Show More bar will appear at the page bottom. Click Show More to see more connections.
+             If you don't have any connection, a simple text is displated *NO PEOPLE*.
 
-.. _MyConnectionPage.Tabs:
+.. _People.Page:
```

Comment:
You changed the section's ID, it may be called in other sections, so either you should restore the old id or change it everywhere called

Reply:
Changed.